guppylang/definition/value.py

```diff
@@ -71,8 +71,12 @@ def compile_call(
         graph: Hugr,
         globals: "CompiledGlobals",
         node: AstNode,
-    ) -> list[OutPortV]:
-        """Compiles a call to the function."""
+    ) -> tuple[list[OutPortV], list[OutPortV]]:
```

Comment:
It'd be nice to return a Namedtuple or similar to avoid mixing up the two outputs.

Same with `CustomInoutCallCompiler.compile_with_inouts`